src/vm/eepolicy.cpp

```diff
@@ -817,9 +817,14 @@ void LogInfoForFatalError(UINT exitCode, LPCWSTR pszMessage, LPCWSTR errorSource
     Thread *pThread = GetThread();
     EX_TRY
     {
-        if ((exitCode == (UINT)COR_E_FAILFAST) && (errorSource == NULL))
+        PrintToStdErrA("\nProcess is terminating due to a ");
```

Comment:
What are possible values of `errorSource`? Is the message going to make sense when we print both `FailFast` / `fatal error` and `errorSource` ?

Reply:
After considerable analysis, it seems it is only set via Debug.Assert and Contract.XXX. I'll test.